src/resmom/mom_main.c

```diff
@@ -5008,6 +5006,17 @@ read_config(char *file)
 		 */
 		cpu_raresync();
 #endif	/* MOM_CPUSET */
+
+		if (joinjob_alarm_time == -1)
+			update_joinjob_alarm_time = 1;
```

Comment:
Do you want to use 1 here, or DEFAULT_JOINJOB_ALARM?

Reply:
This is for setting the boolean 'update_joinjob_alarm_time' to 1 and not the 'joinjob_alarm_time' value, to tell pbs_mom to update the joinjob_alarm_time value when execjob_begin hook alarm value changes. This is because at this point, there was no explicit $sister_join_job_alarm given in mom's config file since joinjob_alarm_time is still set to -1 after processing the file inside read_config().

Reply:
Thanks for clarifying.